androidapp/Crisislab/app/src/main/kotlin/com/example/crisislab/LogItem.kt

```diff
@@ -1,6 +1,7 @@
 package com.example.crisislab
 
+// Data class to represent a log item with height and timestamp
```

Comment:
You know full well what my opinion on these kinds of comments is...

Reply:
Fixed, will push soon, need to ask Mr Viggers about when comments are needed